dali/kernels/imgproc/jpeg/jpeg_distortion_gpu.cuh

```diff
@@ -150,6 +212,9 @@ __global__ void ChromaSubsampleDistortion(const SampleDesc *samples,
   const auto &block = blocks[blockIdx.x];
   const auto &sample = samples[block.sample_idx];
 
+  int aligned_end_y = align_up(block.end.y, 1 + vert_subsample);
+  int aligned_end_x = align_up(block.end.x, 1 + horz_subsample);
```

Comment:
Won't it make us processing too much? How this affects the result?

Reply:
We need to process full 8x8 or 16x16 blocks, to be able to do DCT. If the image dimensions are not a multiple of that, we need to sample out of bounds (repeat the border). We never write back the pixels out of bounds, so we are OK.

Reply:
Understood.